okonomiyaki/utils/misc.py

```diff
@@ -71,7 +72,9 @@ def substitute_variables(d, local_vars):
     is substituted by the value found in the `local_vars' dictionary.  Raise
     ValueError for any variables not found in `local_vars'.
 
-    '$' may be escaped by using '$$'
+    Escapes using '$${name}' are ignored here and not translated to '${name}'.
```

Comment:
The template option should be also in `substitute_variables` and passed down to `subtitute_variable`.